src/adr/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,19 @@
+# 1. Record architecture decisions
+
+Date: 2019-10-02
+
+## Status
+
+Accepted
+
+## Context
+
+We need to record the architectural decisions made on this project.
+
+## Decision
+
+We will use Architecture Decision Records, as [described by Michael Nygard](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions).
```

Comment:
could we extract the content of it, we have no idea how long this link will be valid

Reply:
See https://github.com/apache/james-project/pull/166